### PR TITLE
Improving search method to provide language support

### PR DESCRIPTION
**Pull request by adalbueno**

### CODE REVIEW

**Comment by adalbueno** [comment]
@brh55 Added language supported.
All tests passed successfully.

File: index.js [hunk 1]
```diff
@@ -50,13 +50,15 @@ class googleNewsClient {
 		};
 	}
 
-	search(terms, num = 10) {
+	search(terms, num = 10, language = 'en') {
 		assert(typeof terms === 'string', true, 'expected terms to be string');
 		assert(typeof num === 'number', true, 'expected num to be number');
+		assert(typeof language === 'string', true, 'expercted language to be string');
```
Comment (brh55) [inline index.js]: Could you fix this minor typo here "expercted" to "expected"

**Comment by brh55** [comment]
@adalbueno Looks great! Could you just fix that minor typo, then I shall merge.

**Comment by adalbueno** [comment]
Done @brh55!

**Comment by brh55** [comment]
@adalbueno  Thanks so much! 💃

**Pull request merged**